### PR TITLE
Shared Preferences for Settings

### PR DESCRIPTION
**Pull request by ChloeMJM**
Pulled from Samia's branch and implemented shared preferences for the new settings page. 
Changed the "change username" TextView to EditText to allow user entry. 
Change username button, post replies switch, comment replies switch, all notifications switch all now save values in shared preferences that load in OnCreate. 

These changes should also fix the small error we were getting with her last commit regarding a misnamed button.

### CODE REVIEW

**Comment by zalexchen** [comment]
Great work! Thanks for making the changes, should fix all issues with other branch pull request.

File: app/src/main/java/com/example/warriorsocial/ui/login/RegisterActivity.java [hunk 1]
```diff
@@ -19,11 +19,15 @@
...
 import com.google.firebase.auth.AuthResult;
 import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.FirebaseDatabase;
+
+import org.w3c.dom.Text;
 
 public class RegisterActivity extends AppCompatActivity {
 
-    private EditText userName;
+    private EditText userEmail;
```
Comment (zalexchen) [inline app/src/main/java/com/example/warriorsocial/ui/login/RegisterActivity.java:28]: Good refactored name for this EditText: userEmail better represents this.

**Comment by RichSosa28** [comment]
Nice work! Tested this PR and everything worked smoothly and efficiently.

**Comment by rahulsheth1** [comment]
Everything looks great, a lot of progress on the application!

**Pull request merged**